shared/teams/render.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+import * as React from 'react'
+import {ClickableBox, Box, Icon, ScrollView, Text} from '../common-adapters'
+import {globalColors, globalStyles} from '../styles'
+
+import type {Props} from './render'
+
+function TeamsRender(props: Props) {
+  // TODO: Find right icon for "Join a team" button.
+  return (
+    <Box style={{...globalStyles.flexBoxColumn, alignItems: 'center', height: '100%'}}>
+      <Box style={{...globalStyles.flexBoxRow, alignItems: 'center', height: 48}}>
```

Comment:
shoud likely use globalMargins when possible

Reply:
Done. Although I left the heights widths etc. as constants.